src/Goerli-DssSpell.sol

```diff
@@ -86,278 +27,107 @@ contract DssSpellAction is DssAction {
     // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO -q -O - 2>/dev/null)"
     string public constant override description = "Goerli Spell";
 
-    address constant MCD_VEST_DAI = 0x59B1a603cAC9e38EA2AC2C479FFE42Ce48123Fd4;
-    address constant MCD_VEST_MKR = 0x183bE7a75B8b5F35236270b060e95C65D82f5fF9;
-
-    uint256 constant TWO_PCT            = 1000000000627937192491029810;
-    uint256 constant THREE_PT_FIVE_PCT  = 1000000001090862085746321732;
-    uint256 constant FOUR_PT_FIVE_PCT   = 1000000001395766281313196627;
-    uint256 constant FIVE_PCT           = 1000000001547125957863212448;
-    uint256 constant SIX_PCT            = 1000000001847694957439350562;
-    uint256 constant SEVEN_PCT          = 1000000002145441671308778766;
-
-    uint256 constant MILLION = 10 ** 6;
-    uint256 constant WAD     = 10 ** 18;
+    // Many of the settings that change weekly rely on the rate accumulator
+    // described at https://docs.makerdao.com/smart-contract-modules/rates-module
+    // To check this yourself, use the following rate calculation (example 8%):
+    //
+    // $ bc -l <<< 'scale=27; e( l(1.08)/(60 * 60 * 24 * 365) )'
+    //
+    // A table of rates can be found at
+    //    https://ipfs.io/ipfs/QmefQMseb3AiTapiAKKexdKHig8wroKuZbmLtPLv4u2YwW
+    //
+    uint256 constant ONE_PCT_RATE = 1000000000315522921573372069;
+
+    address constant LERP_FAB = 0xbBD821c291c492c40Db2577D9b6E5B1bdAEBD207;
+
+    // Math
+    uint256 constant THOUSAND = 10 ** 3;
+    uint256 constant MILLION  = 10 ** 6;
+    uint256 constant BILLION  = 10 ** 9;
+    uint256 constant RAY      = 10 ** 27;
+
+    address constant GUNI                   = 0xc5D83e829Ecdce4d67645EE1a1317451e0b4c68d;
+    address constant MCD_JOIN_GUNI_A        = 0xFBF4e3bB9B86d24F91Da185E6F4C8D903Fb63C86;
+    address constant MCD_CLIP_GUNI_A        = 0xFb98C5A49eDd0888e85f6d2CCc7695b5202A6B32;
+    address constant MCD_CLIP_CALC_GUNI_A   = 0x4652E3a6b4850a0fE50E60B0ac72aBd74199D973;
+    address constant PIP_GUNI               = 0x7B3a8452eED8Af27EE2c8a9b826CCfd3b7760461;
```

Comment:
As I said in the chat, if the oracle code changed (even if only a `require`) we can still easily redeploy and put the same exact bytecode here.

Reply:
New oracle: https://goerli.etherscan.io/address/0xf953cdebbbf63607eebc556438d86f2e1d47c8aa#code

Reply:
updated the address